src/components/views/PlayWhites.js

```diff
@@ -118,29 +120,28 @@ const PlayWhites = () => {
         else(drawText="No more Cards left to draw today")
     }
 
-    let drawPile = <BaseContainer className="menu container">
+    let drawPile = <CardContainer className="menu container">
         <CardButton className="card whiteCard"
                     disabled={true}>
             {drawText}
         </CardButton>
-    </BaseContainer>
+    </CardContainer>
 
   return (
     <React.Fragment>
-        <div className={"game description"}>
+        <div className={"index div"} style={{marginTop: "2em", textAlign: "center"}}>
```

Comment:
Style not in style class (not sure if this should  be like this)

Reply:
done